dashboard/app/helpers/users_helper.rb

```diff
@@ -183,7 +183,8 @@ def user_summary(user)
               # for now, we don't allow authorized teachers to be "locked"
               if locked && !user.authorized_teacher?
                 progress[level_id] = {
-                  status: LEVEL_STATUS.locked
+                  status: LEVEL_STATUS.locked,
```

Comment:
Is there an explanation somewhere of the difference between status = locked and locked = true?

Reply:
`status = locked` is used to show the lock icon in a bubble, whereas `locked = true` is used to lock a whole lesson on the overview page (that second part is new to this PR, see `progressHelpers` line 73 above)

Reply:
as we discussed on friday, we are aiming to get rid of `status.locked` entirely in the near future.

Reply:
I think the short answer is that I want to remove LEVEL_STATUS.locked entirely in the near future.  After this change, `status` is no longer used for determining whether the lock icon appears for a **lesson**, but it is still used to determine the color and styling for the bubble for each **level** in the lesson.

Note that `locked` is still a property on the **level** -- we are still following the (IMO, hacky) convention that whether a **lesson** is locked depends on the last level in the lesson.  (This convention is encoded in `stageLocked` in progressHelpers.js).

Reply:
Love that we are going to get rid of status = locked in the future, and would advocate for that refactor to result in better alignment with our backend user_level design if possible.

Reply:
Yes!  This work is being tracked in [LP-1865](https://codedotorg.atlassian.net/browse/LP-1865).  Please add comments if you have ideas/suggestions!